zeppelin/converter.py

```diff
@@ -0,0 +1,212 @@
+import sys
+import json
+import re
+import os
+from datetime import datetime
+
+"""
+ZeppelinConverter is a utility to convert Zeppelin raw json into Markdown
+
+"""
+
+
+class ZeppelinConverter:
+
+    MD_EXT = '.md'
+
+    def __init__(self, ifn, ofn, dire, user, date_created, date_updated):
+        self.index = 0
+        self.input_filename = ifn
+        self.output_filename = ofn
+        self.directory = dire
+        self.user = user
+        self.date_created = date_created
+        self.date_updated = date_updated
+        self.out = []
+
+    def build_header(self, text):
+        """ Generate the header for the Markdown file """
+        header = ['---',
+                  'title: ' + text['name'],
+                  'author(s): ' + self.user,
+                  'tags: ',
+                  'created_at: ' + self.date_created,
+                  'updated_at: ' + self.date_updated,
+                  'tldr: ',
+                  'thumbnail: ',
+                  '---']
+
+        self.out = header + self.out
+
+    def build_markdown(self, lang, body):
+        if body is not None:
+            self.out.append(body)
+
+    def build_code(self, lang, body):
+        self.out.append("```" + lang)
+        self.build_markdown(lang, body)
+        self.out.append("```")
+
+    def process_input(self, paragraph):
+        """
+        Parses paragraph for the language of the code
```

Comment:
Please follow the docstrings pep specifications for the format (first line, blank line, long description)